app/code/Magento/Sales/view/adminhtml/templates/items/column/name.phtml

```diff
@@ -28,9 +28,9 @@
                     <?php else : ?>
                         <?php $_option = $block->getFormattedOption($_option['value']); ?>
                         <?php $dots = 'dots' . uniqid(); ?>
-                        <?= $block->escapeHtml($_option['value']) ?><?php if (isset($_option['remainder']) && $_option['remainder']) : ?> <span id="<?= /* @noEscape */ $dots; ?>"> ...</span>
+                        <?= $block->escapeHtml($_option['value'], ['a']) ?><?php if (isset($_option['remainder']) && $_option['remainder']) : ?> <span id="<?= /* @noEscape */ $dots; ?>"> ...</span>
```

Comment:
Can you please help me understand how allowing the `a` HTML tag is related to the issue and pull request description (that appears to be about `br` tags)

Reply:
thanks for checking @sivaschenko 
i did same thing as magento doing frontend side
https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Sales/view/frontend/templates/order/items/renderer/default.phtml#L23

I think it is allowed for File type option.

Reply:
https://github.com/magento/magento2/blob/2.3-develop/lib/internal/Magento/Framework/Escaper.php#L76

if i did not pass any tag into allowed tag then it will just change html tag to entity tag using htmlspecialchars
https://www.w3schools.com/php/func_string_htmlspecialchars.asp

Reply:
@sidolov  let me know if you need anything to approve this PR

Reply:
@sunilit42 thanks for the explanation! Should the allowed tags passed to the escape method include `br` tag as well? Or `br` is not escaped anyway if at least one allowed tag is passed?

Reply:
@sivaschenko  br is not escaped anyway if at least one allowed tag is passed : yes

Reply:
@sivaschenko  any update on this?